profiling-ffi/.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[build]
+rustflags = ["-C", "relocation-model=pic"]
```

Comment:
Does this have any performance implications?

Reply:
TBH, I do not know. I just moved this from the bash script